airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination_async/AsyncStreamConsumerTest.java

```diff
@@ -258,25 +259,20 @@ private static void consumeRecords(final AsyncStreamConsumer consumer, final Col
 
   // NOTE: Generates records at chunks of 160 bytes
   @SuppressWarnings("SameParameterValue")
-  private static List<PartialAirbyteMessage> generateRecords(final long targetSizeInBytes) {
```

Comment:
this was previously doing too much.

simplified to only generate AirbyteMessages and split out the logic to generate expected records into the `verifyRecords` function